app/Console/Command/DeletePlace.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Console\Command;
+
+use Broadway\CommandHandling\CommandBus;
+use CultuurNet\UDB3\Event\Commands\UpdateLocation;
+use CultuurNet\UDB3\Event\ReadModel\Relations\EventRelationsRepository;
+use CultuurNet\UDB3\Event\ValueObjects\LocationId;
+use CultuurNet\UDB3\Offer\Commands\DeleteOffer;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+
+/**
+ * @url https://jira.publiq.be/browse/III-6109
+ */
+class DeletePlace extends AbstractCommand
+{
+    private const DRY_RUN = 'dry-run';
+    private const PLACE_UUID = 'place-uuid';
+    private const CANONICAL_UUID = 'canonical-uuid';
+
+    private EventRelationsRepository $eventRelationsRepository;
+
+    public function __construct(
+        CommandBus $commandBus,
+        EventRelationsRepository $eventRelationsRepository
+    ) {
+        parent::__construct($commandBus);
+        $this->eventRelationsRepository = $eventRelationsRepository;
+    }
+
+    public function configure(): void
+    {
+        $this
+            ->setName('place:delete')
+            ->setDescription('Mark one place as deleted, and move all it\'s events to the canonical place')
+            ->addArgument(
+                self::PLACE_UUID,
+                null,
+                'Place uuid to delete.'
+            )
+            ->addArgument(
+                self::CANONICAL_UUID,
+                null,
+                'Canonical place uuid to move all events towards.'
+            )
+            ->addOption(
+                self::DRY_RUN,
+                null,
+                InputOption::VALUE_NONE,
+                'Execute the script as a dry run.'
+            );
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): ?int
+    {
+        $placeUuid = $input->getArgument(self::PLACE_UUID);
+        $canonicalUuid = $input->getArgument(self::CANONICAL_UUID);
+
+        if ($placeUuid === null || $canonicalUuid === null) {
+            $output->writeln('<error>Missing argument, the correct syntax is: place:delete place_uuid_to_delete canonical_place_uuid</error>');
+            return 0;
+        }
+
+        if (!$this->askConfirmation($input, $output)) {
+            return 0;
+        }
+
+        $eventsLocatedAtPlaces = $this->eventRelationsRepository->getEventsLocatedAtPlace($placeUuid);
+
+        foreach ($eventsLocatedAtPlaces as $eventsLocatedAtPlace) {
```

Comment:
Still some confusing names:
- `eventsLocatedAtPlaces` should be `eventsLocatedAtPlace` (I think I also proposed this in my comment)
- `eventsLocatedAtPlace` should `eventLocatedAtPlace`